tests/unit_tests/visualization/test_legend_handlers.py

```diff
@@ -0,0 +1,46 @@
+from simtools.util import names
+from simtools.visualization import legend_handlers as leg_h
+
+
+def test_objects():
+    object_list = [
+        leg_h.EdgePixelObject,
+        leg_h.LSTObject,
+        leg_h.MSTObject,
+        leg_h.MeanRadiusOuterEdgeObject,
+        leg_h.OffPixelObject,
+        leg_h.PixelObject,
+        leg_h.SCTObject,
+        leg_h.SSTObject,
+    ]
+
+    for obj in object_list:
+        instance = obj()
+        assert isinstance(instance, object)
+
+
+def test_handlers(io_handler):
+    handler_list = [
+        leg_h.HexEdgePixelHandler,
+        leg_h.HexOffPixelHandler,
+        leg_h.HexPixelHandler,
+        leg_h.LSTHandler,
+        leg_h.MSTHandler,
+        leg_h.MeanRadiusOuterEdgeHandler,
+        leg_h.SCTHandler,
+        leg_h.SquareEdgePixelHandler,
+        leg_h.SquareOffPixelHandler,
+        leg_h.SquarePixelHandler,
+    ]
+
+    for step, handler in enumerate(handler_list):
+        handler_instance = handler()
+        assert isinstance(handler_instance, object)
```

Comment:
In this case I think we can remove this line since the next assert is more meaningful and this one doesn't add anything.

Reply:
ok, it doesn't seem that important to me either